src/app/components/modules/TopRightMenu.jsx

```diff
@@ -117,7 +118,7 @@ function TopRightMenu({
                 <LinkWithDropdown
                     closeOnClickOutside
                     dropdownPosition="bottom"
-                    dropdownAlignment="right"
+                    dropdownAlignment={alignment || 'right'}
```

Comment:
I noticed nobody ever used the `alignment` prop except me and it was hardcoded as always `"right"`